CHANGELOG.md

```diff
@@ -36,7 +36,7 @@ Attention: The newest changes should be on top -->
 
 ### Changed
 
--
+- BUG: Sideslip Angle and Damping Coefficient Calculation [#729](https://github.com/RocketPy-Team/RocketPy/pull/729)
```

Comment:
it is a bug, should be places in "fixed"

I promise one day someone will implement an "auto changelog" tool. Don't worry